db/schema.rb

```diff
@@ -11,11 +11,22 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160615190645) do
+ActiveRecord::Schema.define(version: 20160628234549) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "bids", force: :cascade do |t|
```

Comment:
I think the Bids table looks good. I think we can use the 'status' field on the Items table rather than adding 'status' to bids also.